frontend/src/libs/components/prompt-generation/libs/mocks/category-prompts.ts

```diff
@@ -0,0 +1,41 @@
+import { PromptCategory } from '../enums/enums.js';
+
+const { CHARACTER, SITUATION, SETTING, PROP, GENRE } = PromptCategory;
+
+const categoryPrompts = {
+  [CHARACTER]: ['Girl', 'Detective', '', 'Alien', 'Wizard', 'Superhero'],
```

Comment:
Same about enum destructuring
```suggestion
  [PromptCategory.CHARACTER]: ['Girl', 'Detective', '', 'Alien', 'Wizard', 'Superhero'],
```